ls1intum/Artemis
# [Enhancement] Programming Assessment enable SCA and Build logs

### Checklist

### Motivation and Context


SCA annotations were not displayed during the manual assessment of programming exercises. In this PR we enable back the build output, which adds the annotations to the code editor. Additionally we enable build logs for manual programming assessments.

### Description

- Add back build output for programming assessment
- Add css to properly display annotations with the inline feedback component
- Remove manual assessment check in when fetching build logs

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create Programming Exercise with SCA, participate and assess it (make also a submission with build failed)
4. Check that SCA annotations are displayed in the gutter of the code editor
5. Add inline feedback and check that the SCA annotations is still displayed
6. When assessing the submission with build failed, the build log should also be displayed in the build output component as well as the error annotation.

See also screenshots below

### Test Coverage




### Screenshots


Build failed:
<img width="1660" alt="image" src="https://user-images.githubusercontent.com/41108487/107104172-c77c8200-6820-11eb-99e5-41df3ec5d298.png">

<img width="1665" alt="image" src="https://user-images.githubusercontent.com/41108487/107104204-ea0e9b00-6820-11eb-97e6-b3a18f9d2202.png">


<img width="1669" alt="image" src="https://user-images.githubusercontent.com/41108487/107104189-de22d900-6820-11eb-9090-6139a4738922.png">


SCA:
<img width="1655" alt="image" src="https://user-images.githubusercontent.com/41108487/107104241-0b6f8700-6821-11eb-8c42-0c32a262ee70.png">

<img width="1652" alt="image" src="https://user-images.githubusercontent.com/41108487/107104247-132f2b80-6821-11eb-89ed-b6e0909e4bdb.png">

<img width="1665" alt="image" src="https://user-images.githubusercontent.com/41108487/107104260-23470b00-6821-11eb-817c-2acafeac39bd.png">
